data/account/participationRegistry.go

```diff
@@ -312,11 +317,14 @@ type participationDB struct {
 	store db.Pair
 	mutex deadlock.RWMutex
 
-	writeQueue chan partDBWriteRecord
-	asyncErr   error
+	writeQueue     chan partDBWriteRecord
+	asyncErr       error
+	writeQueueDone chan struct{}
 
 	flushDone      *sync.Cond
 	flushesPending int
+
+	Timeout time.Duration
```

Comment:
`Timeout` -> `flushTimeout` ( it doesn't look like there is a reason to make it public, right ? )

Reply:
It's not used, but I was thinking it might make sense to have it be configurable. I don't mind making it private.